src/Sulu/Bundle/SecurityBundle/EventListener/SuluSecurityListener.php

```diff
@@ -41,8 +41,18 @@ public function __construct(SecurityCheckerInterface $securityChecker)
      */
     public function onKernelController(FilterControllerEvent $event)
     {
-        $controllerDefinition = $event->getController();
-        $controller = $controllerDefinition[0];
+        $controller = $event->getController();
```

Comment:
looks like this never worked with a callable like `MyClass::myCallbackMethod`?
https://www.php.net/manual/de/language.types.callable.php

Reply:
@nnatter oh thats interesting 🙈

Reply:
@nnatter any way to reproduce it? We definitly need a callable which has an instance of `$controller` here as we call `$controller->getLocale()` in this function.
Atleast the listener will not throw an error if a string is given, but you will not be able to use the `SecuredControllerInterface` on your controller then.